packages/eslint-plugin-rune/src/index.ts

```diff
@@ -23,34 +23,67 @@ const restrictedSyntaxBase = [
   },
 ]
 
+const restrictedGlobals = [
+  "exports",
+  "global",
+  "module",
+  "require",
+  "constructor",
+  {
+    name: "Date",
+    message: "Please use Rune.gameTimeInSeconds() for time",
+  },
+  "decodeURI",
+  "decodeURIComponent",
+  "encodeURI",
+  "encodeURIComponent",
+  "escape",
+  "eval",
+  "EvalError",
+  "Function",
+  "RegExp",
+  "toLocaleString",
+  "unescape",
+  "ArrayBuffer",
+  "DataView",
+  "Float32Array",
+  "Float64Array",
+  "Int16Array",
+  "Int32Array",
+  "Int8Array",
+  "Promise",
+  "Proxy",
+  "Reflect",
+  "Symbol",
+  "Uint16Array",
+  "Uint32Array",
+  "Uint8Array",
+  "Uint8ClampedArray",
+  "WeakMap",
+  "WeakSet",
+  "Atomics",
+  "SharedArrayBuffer",
+  "BigInt",
+  "BigInt64Array",
+  "BigUint64Array",
+  "globalThis",
+  "FinalizationRegistry",
+  "WeakRef",
+  "Performance",
+  "Intl",
+]
+
 const logicConfig: ESLint.ConfigData = {
   plugins: ["rune"],
   parserOptions: {
     ecmaVersion: 2021,
   },
   globals: {
```

Comment:
Would be great with a comment explaining why there's both this and `restrictedGlobals` at the top and why these are not part of `restrictedGlobals` :)

Reply:
Sounds great 👍